CHANGELOG.unreleased.md

```diff
@@ -9,6 +9,14 @@
 
 ### Enhancements
 
+**XO 6**:
+
+- [Console]: Displays a loader when the console is loading (PR [#8226](https://github.com/vatesfr/xen-orchestra/pull/8226))
+
+**XO Lite**:
+
+- [Console]: Displays a loader when the console is loading (PR [#8226](https://github.com/vatesfr/xen-orchestra/pull/8226))
```

Comment:
XOLite has its own release process, so it has a different changelog (`@xen-orchestra/lite/CHANGELOG.md`)
And from what is see, XOLite uses `RemoteConsole` and not `VtsRemoteConsole` so this changes will not be visible in XOLite.

```suggestion
```